edgelet/contrib/config/linux/config.yaml

```diff
@@ -32,14 +54,14 @@
 #                       provisioning devices configured for symmetric key
 #                       attestation. Device specific symmetric key.
 #     identity_cert   - Optional. The Edge device identity X.509 certificate
-#                       entry should only be specified when provisioning
-#                       an Edge device configured for X.509 attestation.
+#                       entry should only be specified when an Edge device
```

Comment:
This is a huge nit, but can we lowercase the `Edge`?

Reply:
Resolving per our convo :).